observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -45,6 +45,11 @@ For more information, see xref:../../observability/network_observability/observi
 
 [id="network-observability-cli-1-8_{context}"]
 ==== Network Observability CLI
+
+[IMPORTANT]
+===
+The Network Observability 1.8 CLI updates are unavailable at this time. This is expected to resolve soon.
+===
```

Comment:
Network Observability 1.8 CLI or Network Observability Operator 1.8 CLI? 

Just confirming! I am ignorant and this could be a useless question.

Reply:
```suggestion
[IMPORTANT]
====
The Network Observability 1.8 CLI updates are unavailable at this time. This issue is expected to resolve soon.
====

```

This is not rendering correctly at the moment. I think that should fix it.

Added a word to avoid an ambiguous pronoun.

Reply:
that's a good question, RE Operator. The CLI is just Network observability CLI. It has some of the same functionality of the Operator but it is not the Operator. Its more like a CLI component that accompanies the Operator to make a "Network Observability tool"

Thanks for the word, good idea.